app/src/utilities/logginManager.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict
+import json
+from flask import session
+from app.src.models.User import User
+from app.src.utilities.decorators import template_function
+
+
+def login_user(user: User) -> None:
```

Comment:
if you want to have separate method for log in then it should be responsible for checking password and username

Reply:
that method only create session

Reply:
and im saying thats bad, you said you were studying flask/fastapi code, show me if they do the same

Reply:
added todo, I will change it later. done